README.md

```diff
@@ -52,74 +52,93 @@ const pow = createPow({ host })
 Many APIs are immediately available and don't require authorization.
 
 ```typescript
-const { status, messageList } = await pow.health.check()
+import { POW } from "@textile/powergate-client"
 
-const { peersList } = await pow.net.peers()
+async function getPeersList (pow: POW) {
+  const { status, messagesList } = await pow.health.check()
```

Comment:
docstrings that have await types need to be wrapped in async functions, since there is no await at the top level

Reply:
Would it work if you don't do the whole `import { POW } from "@textile/powergate-client"` and `async function getPeersList (pow: POW)` thing, and instead just make each code snippet section a stand alone example, wrapping all the calls to `pow` in an async function? The way it is now, each code snippet kinda looks like it's supposed to be stand alone because of the import at the top, but the `pow` instance isn't created in the same snippet.

Reply:
As an example, each code snippet could be:
```
import { createPow } from "@textile/powergate-client"

const pow = createPow({ host: "http://0.0.0.0:6002" })

async function exampleCode () {
    const { status, messagesList } = await pow.health.check()
    etc...
}
```

Reply:
For the type checking stuff, you may have to do like I did in that example PR so that `createPow` actually has a return type. Make sense?

Reply:
Just want to explore this idea quickly, but if I'm off base, I'm good with all this and merge away!

Reply:
yeah, i was trying to avoid the whole createPow in every example, but if you prefer it i'll do it that way